src/components/Header/RepoDropdown.tsx

```diff
@@ -8,6 +8,12 @@ export default function RepoDropdown() {
   const [trackedRepos, setTrackedRepos] = useState<GitHubRepo[]>([]);
   const [currentRepo, setCurrentRepo] = useState<GitHubRepo[]>([]);
 
+  const handleRepoClick = (repoName: string) => {
+    setTrackedRepo(repoName);
+    window.dispatchEvent(
+      new CustomEvent('repoChanged', { detail: { repo: repoName } })
```

Comment:
Instead of dispatching custom events, it would perhaps be better to use the best practice of "lifting state up" - https://react.dev/learn/sharing-state-between-components